### ISSUE TITLE
Paper Spigot Pickup Item Issue 1.15.2

### ISSUE THREAD

**Issue by SirGoldman-git** [Zrips/CMI#3898]
**Description of issue or feature request:**
So if you use paper spigot past 355 CMI has an issue where players sometimes cannot pick up items. I tested this on the latest 1.15.2 paper build (384) and the latest CMI build (8.7.5.4). I talked with the paper spigot devs as well and they told me it was an issue with CMI that they can't workaround for the plugin. There is no errors and it has no reason, it just happens to random users. If I revert the paper build before 355 it works fine again.

**Comment by CloudeLecaw** [comment]
Hm seems to be a only 1.15.2 one... On our 1.16.1 one this Problem does not exit ;/

**Comment by Zrips** [comment]
@SirGoldman-git 
is there any specifics related to an update which broke this? Or maybe dev's mentioned something in particular? As CMI does cancel some of the itempickup events, but those require specific conditions, like a player being in afk, vanish or jailed. So something was changed which changed entire behavior for some part of code.

**Comment by SirGoldman-git** [comment]
Sadly, they said something along the lines of "it's a none open source, paid plugin so we can't help". However, I've been testing the latest CMI 8.7.5.4 version with the lastest paper 1.15.2 build and it hasn't happened yet. I only attempted that CMI version when I posted this, at the time I thought it was just going to happen again, sorry about that. So for now, I think this is resolved, I'll let you know if it happens again, seems it was only an issue with a lower version of CMI with that 355+ build version.
